management/ia/rules_test.go

```diff
@@ -56,7 +56,17 @@ func TestRulesAPI(t *testing.T) {
 		})
 
 		t.Run("builtin_template", func(t *testing.T) {
-			params := createAlertRuleParams("pmm_mongodb_restarted", channelID, dummyFilter)
+			params := createAlertRuleParams("pmm_mongodb_restarted", channelID, "threshold", dummyFilter)
+			rule, err := client.CreateAlertRule(params)
+			require.NoError(t, err)
+			defer deleteRule(t, client, rule.Payload.RuleID)
+
+			assert.NotEmpty(t, rule.Payload.RuleID)
+		})
+
+		t.Run("use default value for parameter", func(t *testing.T) {
```

Comment:
Can we check that default value is set?

Reply:
We checking all parameters in `list` tests, line 358. Otherwise, `create` and `list` tests will be identical.

Reply:
@BupycHuk do you want me to copy-paste assertions from `list` test?

Reply:
@BupycHuk ping, last unresolved issue ^

Reply:
ping me in slack if it's urgent, I might miss emails :)

Reply:
And please use `re-request review` button if something required from me.